packages/wrangler/src/dev.tsx

```diff
@@ -467,11 +467,13 @@ export async function startDev(args: StartDevOptions) {
 					accountId={configParam.account_id || getAccountFromCache()?.id}
 					assetPaths={assetPaths}
 					assetsConfig={configParam.assets}
-					port={args.port || configParam.dev.port || (await getLocalPort())}
-					ip={args.ip || configParam.dev.ip}
+					initialPort={
+						args.port ?? configParam.dev.port ?? (await getLocalPort())
```

Comment:
I'm going to come back and refactor `getLocalPort` and `waitForPortToBeAvailable` so they're no longer race-y in a subsequent PR.